Documentation/LanguageServer/FAQ.md

```diff
@@ -1,5 +1,31 @@
 # Frequently asked questions
 
+[Why do I have red underlines everywhere after updating to the latest version](#why-do-i-have-red-underlines-everywhere-after-updating-to-the latest-version)?
```

Comment:
not sure why this link doesn't resolve in the preview. maybe it needs to be committed first?